fe/fe-core/src/main/java/org/apache/doris/load/ExportJob.java

```diff
@@ -356,25 +374,191 @@ private void generateQueryStmt() throws UserException {
         if (LOG.isDebugEnabled()) {
             for (int i = 0; i < selectStmtListPerParallel.size(); ++i) {
                 LOG.debug("ExportTaskExecutor {} is responsible for outfile:", i);
-                for (SelectStmt outfile : selectStmtListPerParallel.get(i)) {
+                for (StatementBase outfile : selectStmtListPerParallel.get(i)) {
                     LOG.debug("outfile sql: [{}]", outfile.toSql());
                 }
             }
         }
     }
 
-    private ArrayList<ArrayList<TableRef>> getTableRefListPerParallel() throws UserException {
-        ArrayList<ArrayList<Long>> tabletsListPerParallel = splitTablets();
+    private void generateLogicalPlanAdapter() throws UserException {
+        // generate 'select...into outfile' sql
+        generateOutfileSqlPerParallel();
+        // debug outfile sql
+        if (LOG.isDebugEnabled()) {
+            for (int i = 0; i < outfileSqlPerParallel.size(); ++i) {
+                LOG.debug("ExportTaskExecutor {} is responsible for outfile sql:", i);
+                for (String outfileSql : outfileSqlPerParallel.get(i)) {
+                    LOG.debug("outfile sql: [{}]", outfileSql);
+                }
+            }
+        }
+
+        outfileSqlPerParallel.stream().forEach(outfileSqlList -> {
+            List<StatementBase> logicalPlanAdapterList = Lists.newArrayList();
+            outfileSqlList.stream().forEach(outfileSql -> {
```

Comment:
Looks like this is a common method to parse a string to a comment.
I suggest to extract it a a Util class, so that others can reuse it.

Reply:
done